Docker/install.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+#########################################
+##        ENVIRONMENTAL CONFIG         ##
+#########################################
+
+# Configure user squeezeboxserver
+export DEBIAN_FRONTEND="noninteractive"
+useradd squeezeboxserver
+usermod -u 99 squeezeboxserver
+usermod -g 100 squeezeboxserver
+usermod -d /home squeezeboxserver
+chown -R squeezeboxserver:users /home
+
+#########################################
+##  FILES, SERVICES AND CONFIGURATION  ##
+#########################################
+
+# Allow acces to /dev/snd
+usermod -a -G audio squeezeboxserver
+
+#########################################
+##             INSTALLATION            ##
+#########################################
+
+# Install LMS
+OUT=$(curl -skL "http://downloads.slimdevices.com/nightly/index.php?ver=8.0")
+
+# Check architecture
+MACHINE_TYPE=`uname -m`
+#X64
+if [ ${MACHINE_TYPE} == 'x86_64' ]; then
+  # Try to catch the link or die
+  REGEX=".*href=\".(.*)amd64.deb\""
+  if [[ ${OUT} =~ ${REGEX} ]]; then
+    URL="http://downloads.slimdevices.com/nightly${BASH_REMATCH[1]}amd64.deb"
+  else
+    exit 1
+  fi
+#X86
+elif [ ${MACHINE_TYPE} == 'i386' -o ${MACHINE_TYPE} == 'i686']; then
+  # Try to catch the link or die
+  REGEX=".*href=\".(.*)i386.deb\""
+  if [[ ${OUT} =~ ${REGEX} ]]; then
+    URL="http://downloads.slimdevices.com/nightly${BASH_REMATCH[1]}i386.deb"
+  else
+    exit 1
+  fi
```

Comment:
Would there even be Docker for i386?...

I could also provide a -docker.deb (or tarball) with just the two supported platforms. And if we fix the base image, then the Perl version is known, too. Which means that the LMS package would be much smaller than the regular packages.

Reply:
Not sure but i've added it as you have the build for that on the downloads page.

Reply:
According to this list i386 is not officially supported by Docker:

https://github.com/docker-library/official-images#architectures-other-than-amd64